client/realm_management.go

```diff
@@ -118,8 +119,12 @@ type InstallInterfaceRequest struct {
 }
 
 // InstallInterface builds a request to install a new major version of an Interface into the Realm.
-func (c *Client) InstallInterface(realm string, interfacePayload interfaces.AstarteInterface) (AstarteRequest, error) {
+func (c *Client) InstallInterface(realm string, interfacePayload interfaces.AstarteInterface, isAsync bool) (AstarteRequest, error) {
 	callURL := makeURL(c.realmManagementURL, "/v1/%s/interfaces", realm)
+
+	query := map[string]string{"async_operation": strconv.FormatBool(isAsync)}
+	callURL = setupURLQuery(callURL, query)
```

Comment:
Maybe we could set the query only if async is true (same for UpdateInterface).
The default value is `false`, so in that case we can omit it.

Reply:
I see. I will update.

Reply:
Updated.